src/main/webapp/app/exercises/modeling/manage/modeling-exercise-update.component.html

```diff
@@ -261,14 +261,17 @@ <h3 jhiTranslate="artemisApp.exercise.sections.grading" id="artemisApp.exercise.
             </div>
         }
         @if (!isExamMode) {
-            <jhi-exercise-feedback-suggestion-options [exercise]="modelingExercise" [dueDate]="modelingExercise.dueDate" />
```

Comment:
Deleting this line is incorrect as it deletes the option to configure Athena's feedback suggestions for modeling exercises. The above checkmark regarding automatic assessment refers to Compass, which is another way to generate feedback suggestions for tutors.
This should be removed in the future.

The UI should look like this:
<img width="1499" alt="image" src="https://github.com/user-attachments/assets/c14e87b1-006c-48f4-9d61-359cf22aadd7" />


But it looks like this currently:
![image](https://github.com/user-attachments/assets/30f21527-cfb7-4599-b721-577135850ef7)